TESTS/usb_device/basic/USBEndpointTester.cpp

```diff
@@ -863,3 +865,4 @@ void USBEndpointTester::start_ep_in_abort_test()
     write_start(_endpoints[EP_INT_IN], _endpoint_buffs[EP_INT_IN], (*_endpoint_configs)[EP_INT_IN].max_packet);
 }
 #endif
+#endif //USB_DEVICE_TESTS
```

Comment:
Lack of new line at the file end, Applies to all files

Reply:
fixed